crates/e2e/src/setup/onchain_components/mod.rs

```diff
@@ -90,6 +90,38 @@ where
     }
 }
 
+pub mod alloy_hook {
```

Comment:
Unless there are good reasons against it I think moving all the alloy specific code to separate files instead of creating ad hoc modules in existing files is probably nicer to work with going forward.